### ISSUE TITLE
Inelegant fail if SIMBAD name resolution fails

### ISSUE THREAD

**Issue by PBarmby** [mshubat/galaxy_data_mines#1]
The call to Simbad.query_region doesn't work if the resolution from object name to coordinates fails (eg because the name server times out). I'm working on a patch.

**Comment by mshubat** [comment]
Great!

**Comment by PBarmby** [comment]
A related but more serious issue (because it could affect the comparison results) is that the query_region functions in astroquery.ned and astroquery.simbad don't work quite the same way when querying by name. This could lead to the NED and SIMBAD queries covering different regions of sky. I think the solution here is to resolve the object name (turn it into coordinates) before doing either query. Working on this now.

**Comment by mshubat** [comment]
I agree that the name should be resolved to coordinates first to try and eliminate any ambiguity to the query regions location.

I don't think that querying byname the way it was implemented should mess with the comparison results too much though, other than not getting as many overlapping objects overall. The symmetric_match_sky_coords_v2 function hopefully should not be allowing any comparisons which are in different locations anyway.. 

Not sure if that was helpful or confusing. But anywho I am still on board with the change. Just need to sort out [#6](https://github.com/mshubat/galaxy_data_mines/issues/6) and we should be in good shape. 👍
